receiver/scraperhelper/scrapercontroller.go

```diff
@@ -210,9 +210,10 @@ func (sc *controller) scrapeMetricsAndReport(ctx context.Context) {
 
 	_, dataPointCount := metrics.MetricAndDataPointCount()
 
-	ctx = obsreport.StartMetricsReceiveOp(ctx, sc.id, "")
+	rec := obsreport.NewReceiver(obsreport.ReceiverSettings{ReceiverID: sc.id, Transport: ""})
```

Comment:
This can be cached in the controller struct.

Reply:
Done in latest commit